diff --git a/rust/revault_lockbox_api/src/file_format/candidate_files/dense_catalogue/overflow.rs b/rust/revault_lockbox_api/src/file_format/candidate_files/dense_catalogue/overflow.rs
index 0ca3c848..5916ef40 100644
--- a/rust/revault_lockbox_api/src/file_format/candidate_files/dense_catalogue/overflow.rs
+++ b/rust/revault_lockbox_api/src/file_format/candidate_files/dense_catalogue/overflow.rs
@@ -159,7 +159,7 @@ impl Catalogue {
                         .checked_add(entry.key.len())
                         .ok_or(Error::CorruptRecord)?;
                     files.push(File {
-                        path: entry.key.clone(),
+                        path: entry.key,
                         permissions: u32::from_le_bytes(entry.value[..4].try_into().unwrap()),
                         info: FileInfo::decode(&entry.value[4..], codec)?,
                         fragments: Vec::new(),
@@ -224,7 +224,7 @@ impl Catalogue {
                         _ => return Err(Error::CorruptRecord),
                     };
                     nodes.push(nodes::Node {
-                        path: entry.key.clone(),
+                        path: entry.key,
                         permissions: u32::from_le_bytes(entry.value[1..5].try_into().unwrap()),
                         target,
                     });
